cql3/expr/prepare_expr.cc

```diff
@@ -127,10 +127,8 @@ usertype_constructor_prepare_expression(const usertype_constructor& u, data_dict
     for (size_t i = 0; i < ut->size(); ++i) {
         auto&& field = column_identifier(to_bytes(ut->field_name(i)), utf8_type);
         auto iraw = u.elements.find(field);
-        expression raw;
-        if (iraw == u.elements.end()) {
-            raw = expr::make_untyped_null();
-        } else {
+        expression raw = expr::make_untyped_null();
+        if (iraw != u.elements.end()) {
```

Comment:
I think you can also make this slightly more efficient (and harder to read) with

```python
expression raw = (iraw == u.elements.end())  ? expr::make_untyped_null() : (++found_values, iraw->second)
```

Reply:
This is the prepare path, so a few instructions here or there don't matter.